shim/tests/src/test_handle_vcpu_kvm_get_fpu.cpp

```diff
@@ -24,13 +24,21 @@
 
 #include "../../include/handle_vcpu_kvm_get_fpu.h"
 
+#include <helpers.hpp>
 #include <kvm_fpu.h>
 #include <mv_types.h>
+#include <shim_vcpu_t.h>
 
+#include <bsl/convert.hpp>
+#include <bsl/safe_integral.hpp>
 #include <bsl/ut.hpp>
 
 namespace shim
 {
+
+    //constexpr auto VAL32{42_u32};
+    constexpr auto VAL64{42_u64};
```

Comment:
remove the global variable. there is no need for this, and they should not be used